tasks/db_deploy/migrations/migrate_versions_4_to_5/migrate_sql.py

```diff
@@ -0,0 +1,311 @@
+    """
+    Full SQL for creating the reconcile_status table. 
+
+    Returns:
+        (sqlalchemy.sql.element.TextClause): SQL for creating reconcile_status table.
+    """
+    return text(
+        """
+        -- Create table
+        CREATE TABLE IF NOT EXISTS reconcile_status  
+        ( 
+          id   	int2 NOT NULL
+        , value	text NOT NULL
+        , CONSTRAINT PK_reconcile_status PRIMARY KEY(id)
+        , CONSTRAINT UNIQUE_reconcile_status_value UNIQUE (value)
+        );
+
+        -- Comments
+        COMMENT ON TABLE reconcile_status 
+            IS 'Reference table for valid status values and status order.';
+        COMMENT ON COLUMN reconcile_status.id 
+            IS 'Status ID';
+        COMMENT ON COLUMN reconcile_status.value 
+            IS 'Human readable status value';
+
+        -- Upsert the data lookup rows for the table
+        INSERT INTO reconcile_status VALUES (1, 'getting S3 list')
+            ON CONFLICT (id) DO NOTHING;
+        INSERT INTO reconcile_status VALUES (2, 'staged')
+            ON CONFLICT (id) DO NOTHING;
+        INSERT INTO reconcile_status VALUES (3, 'generating reports')
+            ON CONFLICT (id) DO NOTHING;
+        INSERT INTO reconcile_status VALUES (4, 'error')
+            ON CONFLICT (id) DO NOTHING;
+        INSERT INTO reconcile_status VALUES (5, 'success')
+            ON CONFLICT (id) DO NOTHING;
+        """
+    )
+
+
+def reconcile_job_table_sql() -> TextClause:
+    """
+    Full SQL for creating the reconcile_job table. 
+
+    Returns:
+        (sqlalchemy.sql.element.TextClause): SQL for creating reconcile_job table.
+    """
+    return text(
+        """
+        -- Create reconcile_job table
+        CREATE TABLE IF NOT EXISTS reconcile_job  
+        ( 
+          id                     	BIGSERIAL NOT NULL
+        , orca_archive_location  	text NOT NULL
+        , status_id              	int2 NOT NULL
+        , inventory_creation_time	timestamp with time zone NOT NULL
+        , start_time             	timestamp with time zone NOT NULL
+        , last_update            	timestamp with time zone NOT NULL
+        , end_time               	timestamp with time zone NULL
+        , error_message          	text NULL
+        , CONSTRAINT PK_reconcile_job PRIMARY KEY(id)
+        , CONSTRAINT FK_reconcile_job_status FOREIGN KEY(status_id) REFERENCES reconcile_status(id)
+        );
+
+        -- Comments
+        COMMENT ON TABLE reconcile_job IS 'Manages internal reconciliation job information.';
+        COMMENT ON COLUMN reconcile_job.id IS 'Job ID unique to each internal reconciliation job.';
+        COMMENT ON COLUMN reconcile_job.orca_archive_location IS 'ORCA S3 Glacier bucket the reconciliation targets.';
+        COMMENT ON COLUMN reconcile_job.status_id IS 'Current status of the job.';
+        COMMENT ON COLUMN reconcile_job.inventory_creation_time IS 'Inventory report creation time from the s3 manifest.';
```

Comment:
Found out after some digging that this is actually when the s3 report was initiated, not created. Their naming isn't great.

Reply:
I just went with "Inventory report initiation time from ..."

Reply:
updated